pkgs/development/libraries/faad2/faad2-hdc-support.patch

```diff
@@ -0,0 +1,644 @@
+From 176e496b2a5461af122e20054079887b0b3d46a8 Mon Sep 17 00:00:00 2001
```

Comment:
Is this an upstream patch to the project?

Reply:
Ah, so this feature is basically derived from this fork? https://github.com/dsvensson/faad2

Reply:
The patch is from the nrsc5 package itself, which pulls from https://github.com/dsvensson/faad2
during the build process and applies the patch. An alternative solution would be
to add a separate package faad2-hdc, which is build from dsvensson faad2 branch.